src/scripts/lsf_status.py

```diff
@@ -48,7 +48,7 @@
 import csv
 
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
-import blah
+# import blah (not used)
```

Comment:
```suggestion
```

Instead of commenting this out, let's just remove the line. With version control, removal is preferred over commenting things out except for certain cases like code that _should_ be there but it needs to be temporarily disabled.